src/test/java/edu/kis/powp/jobs2d/events/SelectMonitoringDeviceTestFigureOptionListener.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.drivers.adapter.UsageMonitoringDriver;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectMonitoringDeviceTestFigureOptionListener implements ActionListener {
+	private final DriverManager driverManager;
+
+	public SelectMonitoringDeviceTestFigureOptionListener(DriverManager driverManager) {
+		this.driverManager = driverManager;
+	}
+	@Override
+	public void actionPerformed(ActionEvent e) {
+		Job2dDriver driver = driverManager.getCurrentDriver();
+		new SetPositionCommand(100, 0)
+				.execute(driver);
+		new OperateToCommand(100, 50)
+				.execute(driver);
+		new OperateToCommand(150, 50)
+				.execute(driver);
+		new OperateToCommand(150, 0)
+				.execute(driver);
+		new OperateToCommand(100, 0)
+				.execute(driver);
+		new SetPositionCommand(0, 0)
+				.execute(driver);
```

Comment:
Why do you create commands instead of working on driver directly. I you want to create command make it reusable (e.g., small command factory).